src/main/webapp/resources/js/apis/linker/linker.js

```diff
@@ -1,6 +1,7 @@
 import axios from "axios";
+import { setBaseUrl } from "../../utilities/url-utilities";
 
-const BASE_URL = `ajax/ngs-linker`;
+const BASE_URL = setBaseUrl(`ajax/ngs-linker`);
```

Comment:
Were you having problems with these urls?  Any ajax request gets intercepted before being sent out and updated with the base url.  See line 76 in app.js

Reply:
These ones were not causing any issues but to keep it consistent with the other files I updated these to match those files

Reply:
Ok, I was worried my interceptor was failing, thanks. :+1:

Reply:
Oh yeah no worries about that. It works great!